src/SFA.DAS.ApprenticeFeedback.Database/Tables/FeedbackTransaction.sql

```diff
@@ -5,7 +5,7 @@
 	[ApprenticeFeedbackTargetId] UNIQUEIDENTIFIER, 
```

Comment:
I'd make the apprentice feedback Target Id Not Null

Reply:
I'd suggest we add this to QF-530 but that's already near release. Should we add this as a new task?

Reply:
QF-530 has been closed due to the changes we are already making here: https://github.com/SkillsFundingAgency/das-apprentice-feedback-api/pull/45

Given this ticket is specifically around sorting out and generating Email Transactions, makes sense to keep it all in one place I would think? We don't have any other feedback api releases to drop it into, and 530 isn't complete based on changes in here as well.

So I'd migrate some changes to this table
- make the apprentice feedback target Id not nullable
- change email and first name to NVARCHAR
Then we can encapsulate it all in this PR

Reply:
Changed as above